articles/openshift/oshift-how-publish-routes-on-multiple-networks.md

```diff
@@ -141,7 +141,7 @@ $ oc expose svc application-2
 route "application-2" exposed
 # we can apply the label to expose our route on the secondary router with the following command
 $ oc label route application-2 "router-secondary=true"
-route "application-2" labeled
+route "application-2" labelled
```

Comment:
As this is a code sample, can you confirm that the British English spelling is correct? Or is using American English correct?

Reply:
This is the output of a command so should stay as it is @suehighmoor @gsmith101

Reply:
@koneill94, I've changed it back. Could you approve please?